src/components/text-editor/prosemirror-adapter/plugins/trigger-plugin.ts

```diff
@@ -0,0 +1,352 @@
+// import { ListItem } from 'src/components/list/list-item.types';
+
+import { Node, NodeType, Schema } from 'prosemirror-model';
+import { Plugin, PluginKey, EditorState } from 'prosemirror-state';
+import { ReplaceStep, ReplaceAroundStep } from 'prosemirror-transform';
+import { EditorView } from 'prosemirror-view';
+import { CustomElement } from 'src/interface';
+
+export const triggerPluginKey = new PluginKey('triggerPlugin');
```

Comment:
Do we need to export this?

Reply:
We need to import it into the prosemirror adapter so that we can instantiate the plugin when the text editor view state is created.

Reply:
I'm not really sure we need to export it? See https://github.com/Lundalogik/lime-elements/pull/3268/commits/58d429f27b22620d17184f27041dc0c8f2bac6c6

Reply:
Ah, I should have looked more closely 👀 
I was referring to the create plugin function. 🙈